### PR TITLE
Add docs about new parameters for resources

### PR DESCRIPTION
**Pull request by Bouh**
@D8H 

Related to your addition in : https://github.com/4ian/GDevelop/pull/5559/files

### CODE REVIEW

File: docs/gdevelop5/events/functions/index.md [hunk 1]
```diff
@@ -182,4 +192,11 @@ This parameter enables auto-completion for previously entered values from other
...
-Every parameter with the same identifier name share their values for the autocompletion.
\ No newline at end of file
+Every parameter with the same identifier name share their values for the autocompletion.
+
+### Resource parameter
+
+These parameters can be, an image, an audio file, a JSON, a font, and a bitmap font. This parameter will return the name of the resource visible in the resource tab.
+The returned value will be only available by getting the value in a JavaScript code events with [this method](/gdevelop5/events/js-code/#use-javascript-to-get-the-value-of-a-parameter-of-a-function).
+
+eventsFunctionContext.getArgument("ParameterName")
```
Comment (Bouh) [inline docs/gdevelop5/events/functions/index.md:202]: I'm removing this line, I made a link above.

File: docs/gdevelop5/events/functions/index.md [hunk 2]
```diff
@@ -153,22 +154,31 @@ Functions can be shared across projects (like actions/conditions that are built
...
+| Scene variable              | Scene variable | The value from a scene variable          | Condition, action, expression |
+| Created objects             | Objects list | A list of objects just created             | Condition, action |
+| Image resource              | Text | The name of the image resource                     | The returned value can only be get from a JavaScript code event |
+| Audio resource              | Text | The name of the audio resource                     | The returned value can only be get from a JavaScript code event |
+| JSON resource               | Text | The name of the JSON resource                      | The returned value can only be get from a JavaScript code event |
+| Font resource               | Text | The name of the Font resource                      | The returned value can only be get from a JavaScript code event |
+| Bitmap font resource        | Text | The name of the Bitmap font resource               | The returned value can only be get from a JavaScript code event |
+
```
Comment (Bouh) [inline docs/gdevelop5/events/functions/index.md:181]: I could add a link to **Resource parameter** section at the bottom of the page when its specified the resource are only available in JS code block.

File: docs/gdevelop5/events/functions/index.md [hunk 3]
```diff
@@ -98,10 +98,11 @@ This is a really simple and not really useful example of a function (you could a
 
 #### Other types of parameters
 
-A parameter can be an object, a text, a number and so ones, for texts and numbers there is two expressions that can be used in events.
+A parameter can be an object, a text, a boolean, a number and so ones, for texts, booleans and numbers there is three expressions that can be used in events.
```
Comment (4ian) [inline docs/gdevelop5/events/functions/index.md:101]: The existing sentence was already pretty bad.
```suggestion
A parameter can be an object, a text, a number or some other kind of data (the name of a layer, a boolean, etc...). You can use expressions and conditions to access the parameters passed to a function.
```

File: docs/gdevelop5/events/functions/index.md [hunk 4]
```diff
@@ -153,22 +154,31 @@ Functions can be shared across projects (like actions/conditions that are built
...
+| Yes or No                     | Boolean      | A boolean value                                           | Condition, action |
+| True or False                 | Boolean      | A boolean value                                           | Condition, action |
+| [Object point](/gdevelop5/objects/sprite/edit-points) | Text         | The name of an point from an object | Condition, action, expression |
+| [Object animation](/gdevelop5/objects/sprite/#adding-an-animation) | Text         | The name of an animation from an object | Condition, action, expression |
+| [Identifier](/gdevelop5/events/functions#identifier-parameter) | Text         | Allow the auto-complete to suggest previously used values | Condition, action, expression |
+| Scene variable              | Scene variable | The value from a scene variable          | Condition, action, expression |
+| Created objects             | Objects list | A list of objects just created             | Condition, action |
+| Image resource              | Text | The name of the image resource                     | The returned value can only be get from a JavaScript code event |
```
Comment (4ian) [inline docs/gdevelop5/events/functions/index.md:176]: The value of the parameter can only be read from a JavaScript code event.

File: docs/gdevelop5/events/functions/index.md [hunk 5]
```diff
@@ -98,10 +98,11 @@ This is a really simple and not really useful example of a function (you could a
 
 #### Other types of parameters
 
-A parameter can be an object, a text, a number and so ones, for texts and numbers there is two expressions that can be used in events.
+A parameter can be an object, a text, a boolean, a number and so ones, for texts, booleans and numbers there is three expressions that can be used in events.
 
 Get a number from a parameter: `GetArgumentAsNumber(string)`
 Get a text from a parameter: `GetArgumentAsString(string)`
+Get a boolean from a parameter: `GetArgumentAsBoolean(string)`
```
Comment (4ian) [inline docs/gdevelop5/events/functions/index.md:105]: To remove. Also, take the opportunity to put a * in front of the two previous lines to make a link :)

Comment (Bouh) [inline docs/gdevelop5/events/functions/index.md:105]:
```suggestion
* Get a number from a parameter: `GetArgumentAsNumber(string)`
* Get a text from a parameter: `GetArgumentAsString(string)`
```

File: docs/gdevelop5/events/functions/index.md (hunk 2 above)
Comment (Bouh) [inline docs/gdevelop5/events/functions/index.md:181]:
```suggestion
| [Identifier](/gdevelop5/events/functions#identifier-parameter) | Text         | Allow the auto-complete to suggest previously used values | Condition, action, expression |
| Scene variable              | Scene variable | The value from a scene variable          | Condition, action, expression |
| Created objects             | Objects list | A list of objects just created             | Condition, action |
| Image resource              | Text | The name of the image resource                     | The value of the parameter can only be read from a JavaScript code event. |
| Audio resource              | Text | The name of the audio resource                     | The value of the parameter can only be read from a JavaScript code event. |
| JSON resource               | Text | The name of the JSON resource                      | The value of the parameter can only be read from a JavaScript code event. |
| Font resource               | Text | The name of the Font resource                      | The value of the parameter can only be read from a JavaScript code event. |
| Bitmap font resource        | Text | The name of the Bitmap font resource               | The value of the parameter can only be read from a JavaScript code event. |

```

**Pull request merged**